package.json

```diff
@@ -12,18 +12,29 @@
     "axios": "^1.5.0",
     "dotenv": "^16.5.0",
     "dropbox": "^10.34.0",
+    "eslint-plugin-react-refresh": "^0.4.20",
```

Comment:
What I meant by my first comment is that all dependencies for eslint aren't used during runtime so they should be in devDependencies category.